hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/NamenodeHeartbeatService.java

```diff
@@ -160,35 +167,28 @@ protected void serviceInit(Configuration configuration) throws Exception {
     this.rpcAddress = getRpcAddress(conf, nameserviceId, originalNnId);
 
     // Get the Service RPC address for monitoring
-    this.serviceAddress =
-        DFSUtil.getNamenodeServiceAddr(conf, nameserviceId, originalNnId);
+    this.serviceAddress = DFSUtil.getNamenodeServiceAddr(conf, nameserviceId, originalNnId);
```

Comment:
Let's restrict the format changes to PRs for refactor.
It is very hard to review and see what is the actual change otherwise.
Let's undo all those.